simulation.py

```diff
@@ -744,9 +745,10 @@ def run(self):
 
 
 if __name__ == "__main__":
-    MainApp = QApplication(sys.argv)
-    if __name__ == "__main__":
+    args = call_args()
```

Comment:
Cool! This method and it's placement works well for logging, a general purpose option or service. Thinking ahead to other command line options like setting a breakpoint in the simulator (e.g. "--breakpoint 1000" ) would be challenging because the Simulation Thread has yet  to created. Perhaps the parsing can be done at this level of main, but we'll need to do something like pass in argument/value pairs to the application (MainWindow) so that it can then pass the options forward in-kind? Or is there a way to have the MainWindow do the parsing taking advantage of